.github/actions/check_artifact_size/action.yaml

```diff
@@ -89,16 +103,15 @@ runs:
             for (let file in fileSizeThresholds) {
               console.log(`Checking file size of ${file}.`);
 
-              const downloadFilePath = path.join('${{ steps.download-artifact.outputs.downloadDir }}', file);
+              const fileName = file.split('/').pop();
+              const downloadFilePath = path.join('${{ steps.download-artifact.outputs.downloadDir }}', fileName);
               if (!fs.existsSync(downloadFilePath)) {
-                console.error(`File ${file} was not uploaded to the main branch.`);
-                continue;
+                core.setFailed(`File ${file} was not uploaded to the main branch.`);
```

Comment:
Add a `return;` or keep the `continue;` after this line.

Reply:
core.setFailed will throw exception, we will not need a return; or continue;.